odo/templates/deployment.yaml

```diff
@@ -2,10 +2,10 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: {{ .Chart.Name }}
-  namespace: {{ .Release.Namespace }}
```

Comment:
Did you intend to remove the namespace from this? Not sure what if any effect that would have

Reply:
oh yeah, definitely should NOT remove this